src/maxflow.rs

```diff
@@ -22,7 +22,7 @@ where
         assert!(Cap::zero() <= cap);
         let m = self.pos.len();
         self.pos.push((from, self.g[from].len()));
-        let rev = self.g[to].len() + if from == to { 1 } else { 0 };
+        let rev = self.g[to].len() + ((from == to) as usize);
```

Comment:
Why not:
```suggestion
        let rev = self.g[to].len() + usize::from(from == to);
```

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#bool_to_int_with_if
According to Clippy's explanation, either way of writing is acceptable. I'll change it to `<uint>::from()`.